docker/build.sh

```diff
@@ -224,7 +222,7 @@ function docker_cmd()
 #            # pushes the juptyer notebooks and rstudio docker images that go on dataproc clusters
 #            bash ./docker/build-tool.sh --push -i jupyter -r "${NOTEBOOK_REPO}" -t "${DOCKER_TAG}" -b "${GIT_BRANCH}"
 #            bash ./docker/build-tool.sh --push -i rstudio -r "${NOTEBOOK_REPO}" -t "${DOCKER_TAG}" -b "${GIT_BRANCH}"
- 
```

Comment:
Wonder if we should delete the above lines and the `build-tool.sh` script. Maybe in a separate PR.